frontend/src/components/globals/UserPanel.vue

```diff
@@ -9,7 +9,7 @@
       <slot>
         <p class="ups">
         <span class="cats">龴ↀ◡ↀ龴 </span>
-        Ups... something went wrong
+        Your cart is empty
```

Comment:
Create new component like UserPanel but name it CartPanel. And use it in your Cart component :+1: